tests/nlu/featurizers/test_count_vectors_featurizer.py

```diff
@@ -806,3 +806,25 @@ def test_use_shared_vocab_exception(
         )
     else:
         new_cvf.train(data)
+
```

Comment:
Can you please add one more test case which fails to train `CountVectorizer` without changes and it passes with changes